File: Add Code Here/PYTHON/BMI_Calculator.py
h=float(input("Enter your height in meters: "))
w=float(input("Enter your Weight in Kg: "))
 
BMI=w/(h*h)
print("BMI Calculated is:  ",BMI)
 
if(BMI>0):
    if(BMI<=16):
        print("You are very underweight")
    elif(BMI<=18.5):
        print("You are underweight")
    elif(BMI<=25):
        print("Congrats! You are Healthy")
    elif(BMI<=30):
        print("You are overweight")
    else: 
        print("You are very overweight")
else:
    print("enter valid details")

File: Add Code Here/C++/BMICalculator.cpp
include<iostream>
using namespace std;

int main()
{

float bmi,weight,height; 
cout<<"Please Enter your weight in Pounds : ";
    cin>>weight;
cout<<"Please Enter your Height in Inches : ";
    cin>>height;
bmi=(weight*703)/(height*height);
cout<<"Your BMI is : "<<bmi<<endl;

if(bmi>25)
    cout<<"Overweight";
else if(bmi<25 && bmi>18.5)
    cout<<"Optimal";
else
    cout<<"Underweight";
return 0;
}
